step-templates/ssis-deploy-ispac-with-enviroment.json

```diff
@@ -0,0 +1,73 @@
+{
+  "Id": "b791c0c2-03ce-40ef-9446-78beb585c0b8",
+  "Name": "SSIS deploy ISPAC With Enviroment",
+  "Description": "Deploy SSIS ISPACs to the SQL Server creating a Enviroment configuration suffix with with the target deployment enviroment",
+  "ActionType": "Octopus.Script",
+  "Version": 17,
```

Comment:
```suggestion
  "Version": 1,
```

Reply:
Have update the new version of the file to version 1